experiment/template/eVOLVER.py

```diff
@@ -482,6 +482,52 @@ def calc_growth_rate(self, vial, gr_start, elapsed_time):
         text_file.write("{0},{1}\n".format(elapsed_time, slope))
         text_file.close()
 
+    def tail_to_np(self, path, window=10):
+        f = open(path, 'rb')
+        if window == 0:
+            return []
+
+        BUFSIZ = 1024
+        f.seek(0, 2)
```

Comment:
You can replace the `2` with `os.SEEK_END`, I think thats a little more clear instead of using the encoded int.